src/components/AdditionalInfo/AdditionalInfo.jsx

```diff
@@ -48,18 +55,21 @@ export default class AdditionalInfo extends React.Component {
     });
 
     const trigger = (
-      <button
-        type="button"
+      <a
+        role="button"
+        id={`${this.expandedContentId}-button`}
```

Comment:
What does setting the `id` give us?

Reply:
This was a remnant from the overcomplicated event listener that I removed - I unfortunately forgot to remove this. This is gone now.